cmd/xl-storage.go

```diff
@@ -2689,7 +2689,7 @@ func (s *xlStorage) StatInfoFile(ctx context.Context, volume, path string, glob
 	}
 	files := []string{pathJoin(volumeDir, path)}
 	if glob {
-		files, err = filepathx.Glob(pathJoin(volumeDir, path))
+		files, err = filepathx.Glob(filepath.Join(volumeDir, path))
```

Comment:
If you add this won't this change the path on Windows if you use filepath, any issue with pathJoin() ?

Reply:
That is fine and it works better with directories.